File: code-coaches/easy/kaleidoscopes/Kaleidoscopes.java
import java.util.Scanner;

public class Kaleidoscopes {

	public static double getTotal(int numKaleidoscopes) {
	    double price = 5.0;
        double tax = 1.07;
        double subtotal = price;

        if (numKaleidoscopes > 1) {
            double discount = 0.9;
            subtotal = (double) numKaleidoscopes * price * discount;
        }

        double total = subtotal * tax;

        // return the total rounded to 2 decimal places
        return Math.round(total * 100.0) / 100.0;
	}

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int numKaleidoscopes = input.nextInt();

		System.out.println(getTotal(numKaleidoscopes));

		input.close();
	}

}

File: code-coaches/easy/kaleidoscopes/kaleidoscopes.py
def get_total(num_kaleidoscopes: int) -> float:
    price = 5
    tax = 1.07

    if num_kaleidoscopes > 1:
        discount = 0.9
        subtotal = num_kaleidoscopes * price * discount
    else:
        subtotal = price

    total = subtotal * tax
    return round(total, 2)


def main():
    num_kaleidoscopes = int(input())
    print(get_total(num_kaleidoscopes))


if __name__ == "__main__":
    main()
